src/test/java/com/epam/qavn/webdriverio/loginPage/LoginTests.java

```diff
@@ -3,24 +3,30 @@
 import com.epam.qavn.core.AbstractTest;
 import com.epam.qavn.pages.LoginPage;
 import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 public class LoginTests extends AbstractTest {
-    LoginPage loginPage = new LoginPage();
+    LoginPage loginPage;
+
+    @BeforeClass
+    public void initData() {
+        loginPage = new LoginPage(driver);
+    }
 
     @Test
     public void invalidLoginTest() {
-        loginPage.tapLoginMenu(driver)
-                .loginWithEmptyAccount(driver);
-        Assert.assertEquals(loginPage.getEmailErrorMessage(driver), "Please enter a valid email address");
-        Assert.assertEquals(loginPage.getPasswordErrorMessage(driver), "Please enter at least 8 characters");
+        loginPage.tapLoginMenu()
+                .loginWithEmptyAccount();
+        Assert.assertEquals(loginPage.getEmailErrorMessage(), "Please enter a valid email address");
+        Assert.assertEquals(loginPage.getPasswordErrorMessage(), "Please enter at least 8 characters");
     }
 
     @Test
     public void validLoginTest() {
-        loginPage.tapLoginMenu(driver)
-                .loginWithValidAccount(driver);
-        Assert.assertEquals(loginPage.getSuccessMessage(driver), "You are logged in!");
-        loginPage.tapOkButton(driver);
+        loginPage.tapLoginMenu()
+                .loginWithValidAccount();
+        Assert.assertEquals(loginPage.getSuccessMessage(), "You are logged in!");
```

Comment:
in case this assertion is failed, the next step will not be executed. we should move it to @After to make sure it always be executed